README.md

```diff
@@ -143,6 +143,12 @@ $ docker run -p 5430:5430 metadata-submitter
 
 Frontend is built and added as static files to backend while building.
 
+## Swagger docs
+
+Swagger UI is already available in the production docker image. During development, you can enable it by executing `bash scripts/swagger/generate.sh`. The script also needs Python 3 and PyYaml to be installed (already available if you installed the development dependencies above).
```

Comment:
Is it ok to run it `./scripts/swagger/generate.sh` when terminal is bash shell? Like an options for the README?

Reply:
Yes, it's ok. I would expect that to be implicit, and wouldn't need to explicitly write that it's possible to run executable scripts by prefixing their path with `./`.

And it doesn't really matter what shell you are running, because of the first line of the script, as long as `bash` is installed, the script should run as expected, at least in a Linux machine.

In the `Dockerfile` it was done like that. https://github.com/CSCfi/metadata-submitter/blob/689d46b84ff1971230625ac50fcc77f21b64de44/Dockerfile